nncf/quantization/algo.py

```diff
@@ -823,6 +826,25 @@ def init_range(self):
                 self._do_range_init(data_loader, num_init_steps, global_init_range_config,
                                     range_init_args.device)
 
+    def run_batchnorm_adaptation(self):
+        init_config = self.quantization_config.get('initializer', {})
+        init_bn_adapt_config = init_config.get('batchnorm_adaptation', {})
+        num_bn_adaptation_steps = init_bn_adapt_config.get('num_bn_adaptation_steps', 200)
+        if num_bn_adaptation_steps < 0:
```

Comment:
probably this checks should be the same for filter pruning

Reply:
Now done for all algos in the base class